### PR TITLE
Linebreak style rule added

### PR DESCRIPTION
**Pull request by initram**
Added new rule to ensure consistent line breaks as discussed in issue #219.

### CODE REVIEW

**Comment by initram** [comment]
Let me know if I should try to squash some of those commits.

File: README.md [hunk 1]
```diff
@@ -470,6 +470,15 @@ This rule is configured with one boolean value:
 
   * boolean -- `true` for enabled rule that forbids the inline styles / `false` for disabled rule that allows them
 
+#### linebreak-style
```
Comment (rwjblue) [inline README.md]: This will need to be moved out into a new `docs/rules/*.md` file.

File: lib/rules/lint-linebreak-style.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+      '  * boolean - `true` to enable (unix style) / `false` to disable\n' +
+      '  * string -- `unix` for LF linebreaks / `windows` for CRLF linebreaks\n' +
+      '\nYou specified `' + JSON.stringify(config) + '`';
+
+    switch (configType) {
+    case 'boolean':
+      // if `true` use `DEFAULT_CONFIG`
+      return config ? DEFAULT_CONFIG : false;
+    case 'string':
+      switch (config){
```
Comment (rwjblue) [inline lib/rules/lint-linebreak-style.js]: small nit-pick: can you add a space between the `)` and `{` chars here?

**Comment by initram** [comment]
I have rebased my changes to the current head of master and have added the spaces you asked for. I am guessing that you can squash the commits when you merge this in. (if you think it is a problem) Otherwise let me know and I will do it.

**Comment by rwjblue** [comment]
Thank you!

**Pull request merged**